provisioning/virtual-machines/ansible/roles/gather-desktop-environment-facts/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# tasks file for gather-desktop-environment-facts
+
+- name: Check whether a Desktop Environment is installed
+  shell: command -v startx || true
+  register: startx_presence_check
+  changed_when: False
```

Comment:
A minor comment: For sake of clarity, that about an appropriate failed_when instead of "|| true"?